projects/ion/src/lib/button/button.component.spec.ts

```diff
@@ -16,3 +16,102 @@ describe('ButtonComponent', () => {
     expect(screen.getByText('configure uma label'));
   });
 });
+
+describe('Primary ButtonComponent', () => {
+  it('should render a button with the ion-btn-primary class when type="primary" is passed', async () => {
+    await render(ButtonComponent, {
+      componentProperties: { type: 'primary' },
+    });
+    const button = screen.findByRole('button');
+    const hasClass = (await button).classList.contains(`ion-btn-primary`);
+    expect(hasClass).toBeTruthy();
+  });
```

Comment:
change to expect to toHaveClass
```suggestion
  it('should render a button with the ion-btn-primary class when type="primary" is passed', async () => {
    await render(ButtonComponent, {
      componentProperties: { type: 'primary' },
    });
    expect(screen.findByRole('button')).ToHaveClass('ion-btn-primary');
  });
```